app/helpers/rapidfire/application_helper.rb

```diff
@@ -10,5 +10,25 @@ def checkbox_checked?(answer, option)
       answers = answer.answer_text.to_s.split(answers_delimiter)
       answers.include?(option)
     end
+
+    def method_missing(method, *args, &block)
```

Comment:
@iffyuva and this is why there is a line `super` inside this `method_missing`. This check will just be injected in the chain of other `method_missing` if there is already one in any classes where helper is included.
Also I am already using this code in production with rails 4.2.